### ISSUE TITLE
Long CLI command issue with containers

### ISSUE THREAD

**Issue by adosztal** [GNS3/gns3-gui#2616]
Original thread: https://gns3.com/community/discussion/network-automation-appliance-cli

If I start a Container and access its console from GNS3, long lines overwrite themselves. If I start the same image on my Linux box, everything works fine (left is the local, right is in GNS3):
![screenshot from 2018-10-31 14-55-43](https://user-images.githubusercontent.com/16632791/47793280-3d2e4500-dd1e-11e8-846d-da175435f183.png)

I'm not sure where the issue is. Does the container's CLI inherit something from my local box that is not passed when it's started from GNS3? Should I add some parameters to the container or the root of the problem is how GNS3 spawns the terminal?

**Comment by grossmj** [comment]
I confirm I can reproduce the bug. This must come from our embedded Telnet server.

**Comment by adosztal** [comment]
I'll check it. Did you just close and open the console of the same node again, or also restarted it, or dragged in a new one?

**Comment by adosztal** [comment]
I checked it, I could reproduce the issue multiple times in all 3 cases.

**Comment by CDWAdamMason** [comment]
Can confirm have the same issue. Very annoying!

**Comment by grossmj** [comment]
I found out I get the problem every time after I resize my console client. I will keep investigating.

**Comment by adosztal** [comment]
That makes sense. My Gnome Terminal profile resizes all opened windows to 100x36; I guess that's why I always encountered the issue.

**Comment by grossmj** [comment]
Yep, actually anything different from 80x24 seems to be a problem.

**Comment by grossmj** [comment]
Here is a quick workaround while I am working on a more permanent solution. Type stty command after the terminal is opened, for example if you open the terminal with the 120x24  size, then type `stty cols 120 rows 24`.

**Comment by CDWAdamMason** [comment]
@grossmj Thank you for the prompt response and solutions. What is the 120x24 size based off of? I am currently working off of vmware workstation and as you may or may not know the screen size is shrunk, does monitor resolution impact the size requirement as specified in your earlier solution?

Also not sure if this is related, but whenever I try to open up a separate terminal window to try and do file management/rw operations while working in the network automation container it simply mirrors the other already opened terminal. I know that this isn't typical functionality as compared to running a regular linux distro. I am using SecureCRT if it matters.
